README.md

```diff
@@ -1,16 +1,162 @@
-# Title
+# Return Values
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify implicit return values in Ruby syntax
+- Recognize the explicit `return` keyword
 
 ## Introduction
 
-## SWBAT 1
+Displaying text in the console is not the only way available to get information
+from our Ruby methods. While `puts` and `print` display information, they don't
+give a _return value_. When writing code, sometimes we'll want our methods to
+evaluate code and _return_ a value -- not just output text, but return it to the
+the place that invoked the method.
 
-## SWBAT 2
+We'll cover the concept of return values in Ruby, and how this actually differs
+from `puts` and `print`commands.
+
+## Identify implicit return values in Ruby syntax
+
+By default, Ruby doesn't display any output, however, this doesn't mean that
+nothing is happening. Methods like `puts` and `print` allow us to output text to
+the console. This is actually different from Ruby's concept of a _return value_.
+
+A _return value_ is the data returned to the program by the execution of a
+method, the assignment of a variable, or other anything in Ruby. Everything has
+a return value!
+
+For example, try pasting these lines into IRB (make sure to include the
+double-quotes around the strings):
+
+| Code                  | Return Value   |
+|-----------------------|----------------|
+| `"Hello world"`       | `"Hello world"`|
+| `6 + 3`               | `9`            |
+| `instructor = "Tim"`  | `"Tim"`        |
+| `total = 5 + 4`       | `9`            |
+| `puts "hello world"`  | `nil`          |
+| `print "hello world"` | `nil`          |
+
+
+As you may notice, Ruby still returns a value after evaluation, even though it
+isn’t explicitly called. It returns the value of the last executed instruction,
+which in these examples, may have been a string declaration, variable
+declaration, or a calculation.
+
+However, when we use `puts` and `print` methods in IRB, you can see the output
+on the screen and a line displayed after it that says `=> nil`. This is because
+`puts` and `print` may print the value you want, but instead of _returning_ that
+value, they display the text but return `nil`. This means that the method has
+not returned a value.
```

Comment:
"Some people think it's strange that `puts` and its friends don't have a return value. But what would that be? That the human at the end of the screen saw the value and paid attention to it? It kinda makes sense, from a certain point of view."